src/store/configureStore.dev.ts

```diff
@@ -75,15 +51,15 @@ export default function configureServiceStore(preloadedState: any, appReducer: a
             if (getAppReducer) {
                 nextAppReducer = getAppReducer(); // eslint-disable-line global-require
             }
-            store.replaceReducer(createDevReducer(baseState, reducerRegistry.getReducers(), nextServiceReducer, nextAppReducer));
+            store.replaceReducer(enableFreezing(createDevReducer(baseState, reducerRegistry.getReducers(), nextServiceReducer, nextAppReducer)));
         });
     }
 
     return store;
 }
 
 function createDevReducer(baseState: any, ...reducers: any) {
-    return enableFreezing(createReducer(baseState, ...reducers));
+    return createReducer(baseState, ...reducers);
```

Comment:
Why move `enableFreezing` out of here? It looks like you're always calling them together anyway.

Reply:
Ooh, good catch -- when I originally removed `createOfflineReducer`, I started getting exceptions around the store being mutated. So to isolate and fix that, I called them in the order `enableFreezing(createOfflineReducer(createDevReducer))` and made sure freezing worked before ripping out `createOfflineReducer` afterwards. The final delta is unnecessary, though -- and in fact `createDevReducer` is largely superfluous at this point.

In fact, I'm not entirely convinced it's necessary to distinguish `.dev.ts` from `.prod.ts` in the first place, given tree-shaking and what not. Either way, I'll clean up this anomaly.